cmd/chisel/cmd_cut.go

```diff
@@ -70,10 +70,15 @@ func (cmd *cmdCut) Execute(args []string) error {
 			Arch:       cmd.Arch,
 			Suites:     archiveInfo.Suites,
 			Components: archiveInfo.Components,
+			Pro:        archiveInfo.Pro,
 			CacheDir:   cache.DefaultDir("chisel"),
 			PubKeys:    archiveInfo.PubKeys,
 		})
 		if err != nil {
+			if err == archive.ErrCredentialsNotFound {
+				logf("Ignoring archive %q (credentials not found)...", archiveName)
```

Comment:
The "ing" gerunds and the "..." are used for actions that will continue until a further point, which is not the case here.  This was an action taken right then: _Archive %q ignored: credentials not found._

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/3adec2115f52d7c843696754c451811025656d48.